pyfive/indexing.py

```diff
@@ -757,7 +757,7 @@ def __init__(self, selection, array):
                     f"expected integer or slice, got {type(dim_sel)!r}"
                 )
 
-            dim_indexer = SliceDimIndexer(slice_, dim_len, dim_chunk_size)
+            dim_indexer = SliceDiImndexer(slice_, dim_len, dim_chunk_size)
```

Comment:
Huh? This looks like a typo?

Reply:
Um, yes! A rogue `CTL-t` in emacs, that switches adjacent letters (very useful!)
```suggestion
            dim_indexer = SliceDimIndexer(slice_, dim_len, dim_chunk_size)
```